test/functional/feature_rbf.py

```diff
@@ -260,33 +240,36 @@ def branch(prevout, initial_value, max_txs, tree_width=5, fee=0.00001 * COIN, _t
             tree_txs = list(branch(tx0_outpoint, initial_nValue, n, fee=fee))
             assert_equal(len(tree_txs), n)
 
-            dbl_tx = CTransaction()
-            dbl_tx.vin = [CTxIn(tx0_outpoint, nSequence=0)]
-            dbl_tx.vout = [CTxOut(initial_nValue - 2 * fee * n, DUMMY_P2WPKH_SCRIPT)]
-            dbl_tx_hex = dbl_tx.serialize().hex()
+            dbl_tx_hex = self.wallet.create_self_transfer(
+                utxo_to_spend=tx0_outpoint,
+                sequence=0,
+                fee=2 * (Decimal(fee) / COIN) * n + Decimal("0.1"),
```

Comment:
Is the last part really needed?
```suggestion
                fee=2 * (Decimal(fee) / COIN) * n,
```

Reply:
Good catch, fixed!